### DISCUSSION TITLE
1.83 no data in visulations

### DISCUSSION THREAD

**Discussion by jgprouty** [akvorado/akvorado#773] [Q&A]
I can login to the webpage on port 8081

when I run  I get nothing back

curl -s http://localhost:8080/api/v0/inlet/flows\?limit=1

I am getting metrics 
curl -s http://localhost:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' 

akvorado_inlet_core_classifier_exporter_cache_size_items 0
akvorado_inlet_core_classifier_interface_cache_size_items 0
akvorado_inlet_core_flows_errors{error="SNMP cache miss",exporter="10.10.11.1"} 8
akvorado_inlet_core_flows_errors{error="sampling rate missing",exporter="10.10.11.1"} 9877
akvorado_inlet_core_flows_http_clients 0
akvorado_inlet_core_flows_received{exporter="10.10.11.1"} 9877
akvorado_inlet_flow_decoder_count{name="netflow"} 1052
akvorado_inlet_flow_decoder_error_count{name="netflow"} 18
akvorado_inlet_flow_decoder_netflow_count{exporter="10.10.11.1",version="9"} 1052
akvorado_inlet_flow_decoder_netflow_errors_count{error="template not found",exporter="10.10.11.1"} 18
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="10.10.11.1",type="DataFlowSet",version="9"} 9877
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="10.10.11.1",type="OptionsDataFlowSet",version="9"} 97
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="10.10.11.1",type="OptionsTemplateFlowSet",version="9"} 97
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="10.10.11.1",type="TemplateFlowSet",version="9"} 2
akvorado_inlet_flow_decoder_netflow_flowset_sum{exporter="10.10.11.1",type="DataFlowSet",version="9"} 953
akvorado_inlet_flow_decoder_netflow_flowset_sum{exporter="10.10.11.1",type="OptionsDataFlowSet",version="9"} 97
akvorado_inlet_flow_decoder_netflow_flowset_sum{exporter="10.10.11.1",type="OptionsTemplateFlowSet",version="9"} 97
akvorado_inlet_flow_decoder_netflow_flowset_sum{exporter="10.10.11.1",type="TemplateFlowSet",version="9"} 2
akvorado_inlet_flow_decoder_netflow_templates_count{exporter="10.10.11.1",obs_domain_id="512",template_id="256",type="template",version="9"} 2
akvorado_inlet_flow_decoder_netflow_templates_count{exporter="10.10.11.1",obs_domain_id="6",template_id="256",type="options_template",version="9"} 97
akvorado_inlet_flow_input_udp_bytes{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1"} 550145
akvorado_inlet_flow_input_udp_in_drops{listener="0.0.0.0:2055",worker="1"} 0
akvorado_inlet_flow_input_udp_packets{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1"} 1070
akvorado_inlet_flow_input_udp_summary_size_bytes{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1",quantile="0.5"} 484
akvorado_inlet_flow_input_udp_summary_size_bytes{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1",quantile="0.9"} 740
akvorado_inlet_flow_input_udp_summary_size_bytes{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1",quantile="0.99"} 1452
akvorado_inlet_flow_input_udp_summary_size_bytes_sum{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1"} 550145
akvorado_inlet_flow_input_udp_summary_size_bytes_count{exporter="10.10.11.1",listener="0.0.0.0:2055",worker="1"} 1070
akvorado_inlet_geoip_db_refresh_total{database="asn"} 1
akvorado_inlet_geoip_db_refresh_total{database="geo"} 1
akvorado_inlet_snmp_cache_expired 0
akvorado_inlet_snmp_cache_hit 19689
akvorado_inlet_snmp_cache_miss 16
akvorado_inlet_snmp_cache_refresh 0
akvorado_inlet_snmp_cache_refresh_runs 8
akvorado_inlet_snmp_cache_size 2
akvorado_inlet_snmp_poller_coalesced_count 16
akvorado_inlet_snmp_poller_pending_requests 0
akvorado_inlet_snmp_poller_seconds{exporter="10.10.11.1",quantile="0.5"} NaN
akvorado_inlet_snmp_poller_seconds{exporter="10.10.11.1",quantile="0.9"} NaN

**Comment by vincentbernat** [comment]
Your equipment does not send the sampling rate. Either configure it to send or configure the sampling rate in Akvorado.
